hammr/commands/migration/migration.py

```diff
@@ -83,12 +83,16 @@ def do_launch(self, args):
 
             migration_config = migration_utils.retrieve_migration_configuration(do_args.file)
             target_format = migration_utils.retrieve_target_format(self.api, self.login, migration_config["target"]["builder"]["type"])
+            image = migration_utils.retrieve_image(migration_config["target"]["builder"], target_format, self.api, self.login)
+            if image is None:
+                return 2
             publish_image = migration_utils.retrieve_publish_image(migration_config["target"]["builder"], target_format)
             if publish_image is None:
                 return 2
             cred_account = migration_utils.retrieve_account(self.api, self.login, migration_config["target"]["builder"]["account"]["name"])
+            publish_image.credAccount = cred_account
```

Comment:
why did you move this line?

Reply:
In fact for me it was not useful to give credAccount as parameter of create_migration method just to set it in publishImage.